executor/delete.go

```diff
@@ -133,6 +133,7 @@ func (e *DeleteExec) deleteSingleTableByChunk(ctx context.Context) error {
 			}
 			rowCount++
 		}
+		chk = e.children[0].newChunkWithCapacity(chk.NumRows())
```

Comment:
If the child operator's initial chunk size is 32, the newly allocated chunk will always be 32 sized?

Reply:
if the child's initial size = 32, it will allocate 32 in the first loop and after `Next()` return chk NumRows will be both `<=32` or  `>32`, so in the 2nd loop will use numRows as capacity if it works as my think